misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt

```diff
@@ -105,6 +109,49 @@ abstract class PathFeatureAssertionsSpec(
         }
     }
 
+    describeFun("val ${resolveFeature.name}") {
```

Comment:
This class seemed to use multiples describeFun as opposed at what I had to do in OptionalAssertionsSpec. I followed the pattern already used.

Reply:
that's fair enough